mlir/test/Conversion/GPUToSPIRV/lookup-target-env.mlir

```diff
@@ -0,0 +1,17 @@
+// RUN: mlir-opt --spirv-attach-target='caps=Shader exts=SPV_KHR_storage_buffer_storage_class' --convert-gpu-to-spirv %s -o - | FileCheck %s
```

Comment:
What do you think about running `convert-gpu-to-spirv` only? In the current form, it's not very clear where the 'targets' attribute is materialized.